tests/features/bootstrap/FeatureContext.php

```diff
@@ -212,4 +212,42 @@ public function createLanguages($langcode) {
     $this->languageCreate((object) ['langcode' => $langcode]);
   }
 
+  /**
+   * Prepare for PHP errors log.
+   *
+   * @BeforeSuite
+   */
```

Comment:
If this is changed to `@BeforeStep` and `@AfterStep` then the failures show up nicely when the test progresses:

```
$ ./behat 
.....................................................F----------.....- 70
.....-.....-.....-.....-.....-.....-.....-.....-.....-.....-.....-.... 140
.-.....-.....-.....-.....-.....-...................................... 210
...................................................................... 280
...................................................................... 350
.......................................................-.............. 420
...................................................................... 490
.....................--...-----...-
```